### PR TITLE
Re-Write Cards Page of the Docs

### PR DESCRIPTION
**Pull request by AlexLakatos**
deploy preview: https://602111663966b2325c06baf7--fidel-website.netlify.app/docs/cards

### CODE REVIEW

File: cards/README.md [hunk 1]
```diff
@@ -1,84 +1,119 @@
...
+
+If you don't want to use our secure and PCI compliant SDKs, you must get PCI Compliant before using our Cards API. Contact us at devrel@fidel.uk for more information.
+
+```sh
+curl -X POST \
+  https://api.fidel.uk/v1/programs/f76ed1be-e434-480b-aa1d-ff48f548f62a/cards
+  -H 'Content-Type: application/json'
+  -H 'Fidel-Key: pk_test_62f02030-0409-4eb5-ab94-6eff05b3d888'
+  -d '{
+  	"number": "4444000000004222",
```
Comment (danitre) [inline cards/README.md:59]: This looks a little bit too on the left, not sure if it will be showed or not on the page

Comment (AlexLakatos) [reply cards/README.md:59]: Screenshot 2021-02-08 at 11 38 37

Comment (AlexLakatos) [reply cards/README.md:59]: That's how it looks on the preview. Is that ok @danitre ?

Comment (danitre) [reply cards/README.md:59]: Yep, all good!

**Review by danitre — APPROVED** [review]
🚀

**Pull request merged**